seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/connection/SimpleJdbcConnectionProvider.java

```diff
@@ -129,9 +129,6 @@ public Connection getOrEstablishConnection()
                 "No suitable driver found for " + jdbcOptions.getUrl(), "08001");
         }
 
-        //Auto commit is used by default
-        connection.setAutoCommit(true);
-
```

Comment:
How can we set the `autocommit` parameter?

Reply:
Set in url

Reply:
However, some database implementations do not have the parameter `autocommit`, such as teradata. please track this https://teradata-docs.s3.amazonaws.com/doc/connectivity/jdbc/reference/current/frameset.html